### PR TITLE
Update Files.de-DE.xlf

### PR DESCRIPTION
**Pull request by R3voA3**
Added translation for new dialog

### CODE REVIEW

File: Files/MultilingualResources/Files.de-DE.xlf [hunk 1]
```diff
@@ -421,29 +421,29 @@
         </trans-unit>
         <trans-unit id="NavigationToolbarNewTab.Text" translate="yes" xml:space="preserve">
           <source>New Tab</source>
-          <target state="new">New Tab</target>
+          <target state="new">Neuer Tab</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.de-DE.xlf]: In order for the translations to work, the state needs to be `Translated`

File: Files/MultilingualResources/Files.de-DE.xlf [hunk 2]
```diff
@@ -421,29 +421,29 @@
...
         </trans-unit>
         <trans-unit id="ExtractFilesDialogDescription.Text" translate="yes" xml:space="preserve">
           <source>Pick a location to extract this compressed archive to. You'll need to stay in the current folder until we're done. A new tab will open up with the extracted items.</source>
-          <target state="new">Pick a location to extract this compressed archive to. You'll need to stay in the current folder until we're done. A new tab will open up with the extracted items.</target>
+          <target state="new">Wähle aus wohin das Archiv entpackt werden soll. Sie müssen im derzeitigen Ordner bleiben bis das Entpacken beended wurde. Ein neuer Tab mit den entpackten Dateien wird anschließend geöffnet.</target>
         </trans-unit>
         <trans-unit id="ExtractFilesDialogBrowseButton.Content" translate="yes" xml:space="preserve">
           <source>Browse</source>
-          <target state="new">Browse</target>
+          <target state="new">Browsen</target><!---Need to see the final dialog to find a better translation-->
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.de-DE.xlf]: The browse button is to choose a different location to extract the files to.

**Review by yair100 — CHANGES_REQUESTED** [review]

**Review by R3voA3 — COMMENTED** [review]
- changed state from new to translated
- changed ExtractFilesDialogBrowseButton.Content translation

**Comment by lukeblevins** [comment]
@R3voA3 Thanks for this! Also, @yair100 has been really great in helping me with the translations

**Review by yair100 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
Thank you for always keeping your translations up to date!

**Pull request merged**